test/unit/UtilsTest.cpp

```diff
@@ -122,8 +122,6 @@ TEST(UtilsTest, test_base64)
 
     EXPECT_EQ(decoded.size(), data.size());
     EXPECT_EQ(size, begin_size);
-
-    EXPECT_THROW(Utils::base64_decode("Thisisatest"), std::runtime_error);
```

Comment:
The title of this PR said it was just formatting, but this test was removed. IDK if that was intentional.